tests/functional/adapter/utils/test_timestamps.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from dbt.tests.adapter.utils import test_timestamps
+
+
+class TestCurrentTimestampSnowflake(test_timestamps.TestCurrentTimestamps):
```

Comment:
At minimum, need the same change as [here](https://github.com/dbt-labs/dbt-snowflake/pull/273/files/837f837bead2b2c1885a6981a908f30844aa2c54..1896507ceb00586b230ec3b76fe7c6fa208b63d0#diff-d73b8b3992c53527ebb2550e03c14ea58330ec87b8d25a79e9204bf08738b561R10-R11) in order to work since there were changes to your branch in dbt-core [here](https://github.com/dbt-labs/dbt-core/pull/5838/files/a8b5bfe25191954484dbb3fa0f7bc9b2f1917ea0..c97cead2f7d251b4a4ca3a9531d5da3a98c944a5).

To follow existing conventions most closely, I think it would be:
```suggestion
from dbt.tests.adapter.utils.test_timestamps import BaseCurrentTimestamps


class TestCurrentTimestampRedshift(BaseCurrentTimestamps):
```